src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockCropSimple.java

```diff
@@ -90,63 +75,27 @@ public boolean onBlockActivated(World worldIn, BlockPos pos, IBlockState state,
             ItemStack foodDrop = crop.getFoodDrop(state.getValue(getStageProperty()));
             if (!foodDrop.isEmpty())
             {
+                ItemStack seedDrop = new ItemStack(ItemSeedsTFC.get(crop));
+                seedDrop.setCount(0);
+                SimpleSkill skill = CapabilityPlayerData.getSkill(playerIn, SkillType.AGRICULTURE);
+
+                if (skill != null)
+                {
+                    foodDrop.setCount(1 + Crop.getSkillFoodBonus(skill, RANDOM));
+                    // omit the +1 because the plant stays alive.
+                    seedDrop.setCount(Crop.getSkillSeedBonus(skill, RANDOM));
```

Comment:
Would be nicer to have
```java
ItemStack seedDrop = ItemStack.EMPTY;
```
and then
```java
seedDrop = new ItemStack(ItemSeedsTFC.get(crop), Crop.getSkillSeedBonus(skill, RANDOM));
```
As `.setCount(0)` is really ugly.

Reply:
Would putting the 0 in the constructor be okay? I feel like `seedDrop` should always be a seed item during the variable's lifespan in case someone needs to change how the amount of seeds dropped is calculated outside the scope of that if statement.